lamindb/curators/__init__.py

```diff
@@ -8,6 +8,7 @@
    MuDataCurator
    SpatialDataCurator
    TiledbsomaExperimentCurator
+   CxGCurator
```

Comment:
I don't think we want a Curator anymore, unless I'm wrong. The idea would be to just provide a schema, no? There wouldn't be any specific curator code for CxG, would it?

Reply:
No, the Curator needs custom code for missing ontology values (such as "normal"), missing defaults, handling names vs ontology_id, ...

I currently don't think that we can hack all of that into a Schema.

I'll first add tests for the low level functionality that I changed/fixed and then I'll open a Slack thread to discuss this topic. Yesterday, I actually had moved a CxG generator function to `lamindb.examples.anndata` just to move it back. I understand what you're looking for.

Reply:
The function that gets the Schema can probably do most of that. I'll revisit this ASAP but first tests!

Reply:
(Sorry that comment was from me)
It should not have its Curator code. We need a different strategy to add control terms, we can just guide users to prepare the instance with a proper notebook.